src/components/atoms/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -41,18 +49,27 @@ export const ConfirmationModal: FC<ConfirmationModalProps> = ({
   const isShown = show !== undefined ? show : isVisible;
 
   return (
-    <Modal show={isShown} onHide={hide}>
+    <Modal
+      className="confirmation-modal"
+      show={isShown}
+      onHide={hide}
+      centered={centered}
+    >
       <Modal.Body>
-        <div className="confirmation-modal">
-          {hasHeader && <h2 className="confirm-header">{header}</h2>}
+        <div>
+          {hasHeader && <h3 className="confirm-header">{header}</h3>}
```

Comment:
Should probably be part of `Modal.Title` without the `h3` that most likely breaks the semantics of h* elements just for the sake of visual appearance

```suggestion
          {hasHeader && <h3 className="ConfirmationModal__header confirm-header">{header}</h3>}
```

Reply:
fixed here: https://github.com/sparkletown/sparkle/pull/2131/commits/9b9f6c1a6dd9bc6c13b69121718d28b857ec4bf4